lang/en.inc.php

```diff
@@ -80,6 +80,7 @@
 $messages['passwordinvalid'] = "Authentication has failed";
 $messages['passwordok'] = "Authentication succeeds!";
 $messages['passwordrefused'] = "Password was refused";
+$messages['passwordinhistory'] = "Password in history";
```

Comment:
could you:
- add all other language with a default value in english
- translate the value for our customer language?